WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainDetailsViewController+LocalizedStrings.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+
+extension RegisterDomainDetailsViewController {
+    enum Localized {
+        enum PrivacySection {
+            static let title = NSLocalizedString(
+                "Privacy Protection",
+                comment: "Register Domain - Privacy Protection section header title"
+            )
+            static let description = NSLocalizedString(
+                "Domain owners have to share contact information in a public database of all domains. With Privacy Protection, we publish our own informations instead of yours and privately forward any comminucation to you",
```

Comment:
There are two typos here:
- "informations" should be "information".
- "comminucation" should be "communication".

Also, please add a period at the end of this sentence.

Reply:
Done

Reply:
Also removed forgotten parts from Domains.storyboard